xml/bp_troubleshooting_tips.xml

```diff
@@ -739,25 +732,33 @@ ID WEIGHT TYPE NAME    UP/DOWN REWEIGHT PRIMARY-AFFINITY
  </sect1>
  <sect1 xml:id="storage-bp-recover-clockskew">
   <title>Fixing Clock Skew Warnings</title>
-
   <para>
-   The time information in all cluster nodes must be synchronized. If a node's
-   time is not fully synchronized, you may get clock skew warnings when
-   checking the state of the cluster.
+   As a general rule, time synchronization must be configured and running on
+   all nodes. Once time synchronization is set up, the clocks should not get
+   skewed. However, if a clock skew is to occur, this is likely due to the
+   <literal>chronyd.service</literal> not running on one or more hosts.
   </para>
-
-  <para>NOTE: Write section on clock skews with cephadm -
-    1. double-check the time_server configs in ceph-salt are good.
-    2. call ceph-salt apply
+  <note>
+    <para>
+      It is also possible that the battery on the motherboard has died, and
+      the clock skew will be more pronounced. If this is the case, be aware
+      that it will take quite some time for chronyd to re-sync the clocks.
+    </para>
+  </note>
+  <para>
+    If you receive a clock skew warning, confirm that the <literal>chronyd.service</literal>
+    is running on all cluster nodes. If not, restart the service and wait for
+    chronyd to re-sync the clock.
   </para>
-
- </sect1>
- <sect1 xml:id="cluster-health">
-  <title>Determining Cluster Health</title>
-
   <para>
-   TBD
+    To do so, review the <command>ceph-salt config ls</command> output and
+    verify that <option>/time_server</option> is not set to <literal>disable</literal>.
   </para>
+  <para>
+    If in the case it is, run <command>ceph-salt apply</command> to completion
+    to reconfigure the <literal>chronyd.service</literal>.
```

Comment:
And this one as well, please. It's enough to tell the user to verify that chronyd.service is running on all the nodes.

I don't think it's feasible for the documentation to detail all possible reasons why a clock skew might occur.

Maybe you could ask the SLE documentation folks if time sync configuration and troubleshooting is covered in the SLE docs? If so, then we could point readers to that: "For more information on troubleshooting issues with time sync, see ... <chapter, verse> ... of the Book of SLE"